cips/cip-25.md

```diff
@@ -17,9 +17,10 @@ This Meta CIP lists the CIPs included in the Ginger network upgrade.
 
 ### Included CIPs
 
+- [CIP-10](./cip-10.md): Coordinated network upgrades
 - [CIP-21](./cip-21.md): Introduce blob type with verified signer
 - [CIP-24](./cip-24.md): Versioned Gas Scheduler Variables
-- CIP-TBD: limiting compute per block
+- CIP-TBD: Reduce block time to six seconds
```

Comment:
AFAIK we may have two CIPs still:
1. limiting compute per block
2. reducing block time to six seconds